src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2238,12 +2254,19 @@ private void printDevModeMessages(boolean inputUnavailable, boolean startup) {
 
         if (!inputUnavailable) {
             // the following will be printed on startup and every time after the tests run
-            if (hotTests) {
-                String message = "Tests will run automatically when changes are detected. You can also press the Enter key to run tests on demand.";
-                info(startup ? formatAttentionMessage(message) : message);
+            if (blockTests) {
+                String message1 = "A classpath error with Maven has been detected. Tests will not run in dev mode on this project.";
+                String message2 = "Run tests outside of dev mode with `mvn verify` or `mvn failsafe:integration-test`.";
+                info(startup ? formatAttentionMessage(message1) : message1);
+                info(startup ? formatAttentionMessage(message2) : message2);
```

Comment:
Suggest to log these as warning or error so that they are more visible, especially since they are part of the dev mode startup messages and users may not notice those otherwise.

Reply:
We've done something similar with devc [here](https://github.com/OpenLiberty/ci.common/blob/main/src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java#L2277) for port warnings, see example:
![Screen Shot 2021-10-20 at 10 56 50 AM](https://user-images.githubusercontent.com/29640130/138118844-48aa04fa-9150-4b6f-8e06-683b3fa0fa9f.png)
I think the formatting being off is fine.  At least having something different there draws attention.

Reply:
Updated to print an error message and removed the error message that displays immediately after the startup message.